node/v2/index.js

```diff
@@ -27,6 +27,7 @@ module.exports.Types = Types;
 
 var RetryFlags = require('../retry-flags.js');
 var Frame = require('./frame');
+var LazyFrame = require('./frame');
```

Comment:
oh man... I wonder if that means that I'm not even doing anything different then ... lol